pages/_app.js

```diff
@@ -1,11 +1,11 @@
-import GlobalStyles from "../styles/GlobalStyles";
+//import GlobalStyles from "../styles/GlobalStyles";
+import Layout from "../components/Layout";
 
 function MyApp({Component, pageProps}) {
   return (
-    <>
-      <GlobalStyles />
```

Comment:
You should keep `<GlobalStyles />` here - otherwise settings in the global stylesheet will have no effect.